<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWeakTitle</class>
 <widget class="QDialog" name="EditWeakTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Weak Title - %1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Area</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="posX">
            <property name="minimum">
             <number>-2147483647</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="posY">
            <property name="minimum">
             <number>-2147483647</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="width">
            <property name="minimum">
             <number>-2147483647</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="height">
            <property name="minimum">
             <number>-2147483647</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Font</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fontPreview">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectFont">
          <property name="text">
           <string>Select font</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Text alignment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="textAlignment">
          <item>
           <property name="text">
            <string>Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Center</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="isVertical">
        <property name="text">
         <string>Vertical</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Label colour</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Weakdays</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="colWeakdayHex">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colWeak">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectWeakdayCol">
          <property name="text">
           <string>Select colour</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Saturday</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="colSatHex">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colSat">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectSatCol">
          <property name="text">
           <string>Select colour</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Sunday</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="colSunHex">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colSun">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectSunCol">
          <property name="text">
           <string>Select colour</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Labels</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QTreeWidget" name="labels">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addItem">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeItem">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>posX</tabstop>
  <tabstop>posY</tabstop>
  <tabstop>width</tabstop>
  <tabstop>height</tabstop>
  <tabstop>fontPreview</tabstop>
  <tabstop>selectFont</tabstop>
  <tabstop>textAlignment</tabstop>
  <tabstop>isVertical</tabstop>
  <tabstop>colWeakdayHex</tabstop>
  <tabstop>selectWeakdayCol</tabstop>
  <tabstop>colSatHex</tabstop>
  <tabstop>selectSatCol</tabstop>
  <tabstop>colSunHex</tabstop>
  <tabstop>selectSunCol</tabstop>
  <tabstop>addItem</tabstop>
  <tabstop>removeItem</tabstop>
  <tabstop>ok</tabstop>
  <tabstop>cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>EditWeakTitle</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>EditWeakTitle</receiver>
   <slot>onAccepted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addItem</sender>
   <signal>clicked()</signal>
   <receiver>EditWeakTitle</receiver>
   <slot>onAddItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeItem</sender>
   <signal>clicked()</signal>
   <receiver>EditWeakTitle</receiver>
   <slot>onRemoveItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectWeakdayCol</sender>
   <signal>clicked()</signal>
   <receiver>EditWeakTitle</receiver>
   <slot>onSelectWeakdayColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectSatCol</sender>
   <signal>clicked()</signal>
   <receiver>EditWeakTitle</receiver>
   <slot>onSelectWeakendColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectSunCol</sender>
   <signal>clicked()</signal>
   <receiver>EditWeakTitle</receiver>
   <slot>onSelectWeakstartColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectFont</sender>
   <signal>clicked()</signal>
   <receiver>EditWeakTitle</receiver>
   <slot>onSelectFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAccepted()</slot>
  <slot>onAddItem()</slot>
  <slot>onEditItem()</slot>
  <slot>onRemoveItem()</slot>
  <slot>onAddChild()</slot>
  <slot>onSelectWeakdayColour()</slot>
  <slot>onSelectWeakendColour()</slot>
  <slot>onSelectWeakstartColour()</slot>
  <slot>onSelectFont()</slot>
 </slots>
</ui>
